src/app/notebox/page.tsx

```
@@ -0,0 +1,75 @@
+import useUserNote from '@/hooks/noteHooks/useUserNotes';
+import {
+  fetchUserInfo,
+  fetchUser,
+  fetchUserWorries
+} from '../utils/supabase/db';
+import NoteCard from '@/components/noteBoxComponents/NoteCard';
+
+const NotePage = () => {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const { data: loginUser } = useUserData();
+  const [notes, setNotes] = useState<any[]>([]);
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)